aladdin-container.sh

```diff
@@ -85,17 +89,11 @@ function environment_init() {
     echo "CLUSTER_CODE = $CLUSTER_CODE"
     echo "NAMESPACE = $NAMESPACE"
 
-    # Kops uses AWS_PROFILE instead of AWS_DEFAULT_PROFILE
-    export AWS_PROFILE="$AWS_DEFAULT_PROFILE"
-
-    # Sanity check the user's aws configuration if init is set
-    if "$INIT" && ! "$IS_LOCAL"; then
-        test_aws_config
-    fi
+    _handle_aws_config
 
     # Make sure we are on local or that cluster has been created before initializing helm, creating namespaces, etc
-    if "$IS_LOCAL" || ( kops export kubecfg --name $CLUSTER_NAME &> /dev/null && \
-        kops validate cluster --name $CLUSTER_NAME ) &> /dev/null; then
+    if "$IS_LOCAL" || ( kops export kubecfg --name $CLUSTER_NAME > /dev/null && \
+        kops validate cluster --name $CLUSTER_NAME ) > /dev/null; then
```

Comment:
While looking into a Jenkins Cluster issue I found myself unable to launch `aladdin bash`, reason being is that `kops validate cluster` was failing (the cluster went down, yikes), and so `_handle_authentication_config` was not being called (below) and so aladdin was exiting with:
```
/root/aladdin/commands/bash/container/bash/bash: line 12: AUTHENTICATION_ENABLED: unbound variable
```